dev/index.html

```diff
@@ -7,7 +7,10 @@
     <div class="container" />
 
     <script>
-      TeamContributionCalendar.default(".container", ["tccemptyuser"])
+      TeamContributionCalendar.default(".container", 
+      [["gomorizsolt", ['2019-07-01','2019-08-04']], "tccemptyuser1"], 
+      [["nkapolcs", ['2019-09-01','2020-01-04']]], 
```

Comment:
This wasn't clarified as a requirement initially but let's support only a start date with an open end date, e.g.

```suggestion
      [{ name: "nkapolcs", from: '2019-09-01' }], 
```

Let's also add an example of this to the readme.

Reply:
> open end date

It means the end date is always the current date, right?

Reply:
Yes

Reply:
My bad. In this example, it means that the end date is current, however we should support the end date option too. Updated the readme with examples.

Reply:
Cool, thanks for the update and clarification. So what this actually means is that:
- if the *start date* is missing but the *end date* is specified then we should check for an upper limit
- if the *end date* is missing but the *start date* is specified then we should check for a lower limit
- if both of them are *missing*, everything remains the same as before
- if both of them are *specified* then we should check for both the upper and lower limit

I've added this logic in cf40207 although the first option isn't mentioned in the examples. Does this clutter the logic or it's fine as-is?

Reply:
> although the first option isn't mentioned in the examples.

9f842d3